tests/miral/config_file.cpp

```diff
@@ -44,12 +44,22 @@ class PendingLoad
     {
         std::unique_lock lock{mutex};
 
-        if (!cv.wait_for(lock, std::chrono::milliseconds{10}, [this] { return !pending_loads; }))
+        if (!cv.wait_for(lock, std::chrono::seconds{1}, [this] { return !pending_loads; }))
```

Comment:
This is still 100x what we had before - I don't think there's that much difference between the test systems. Surely one order of magnitude would suffice